schemas/1.6.0/adaptive-card.json

```diff
@@ -3304,6 +3310,18 @@
           "description": "Label position hint for input.",
           "version": "1.6"
         },
+        "labelWidth": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "type": "number"
+            }
+          ],
+          "description": "Determines the width of the label in percent like 40 or a specific pixel width like '40px' when label is placed inline with the input . labelWidth would be ignored when the label is displayed above the input.",
```

Comment:
For a percentage, would the value be `40` or `40%`?

Reply:
for percentage, value would be a number. So it should be just 40. I think `width` in column also follows similar guidelines https://adaptivecards.io/explorer/Column.html